### ISSUE TITLE
534507E5 - Golden Axe: Beast Rider

### ISSUE THREAD

**Issue by Sleepy93** [xenia-project/game-compatibility#712]
[Marketplace](http://marketplace.xbox.com/en-US/Product/Golden-Axe-Beast-Rider/66acd000-77fe-1000-9115-d802534507e5)

Tested on (https://github.com/benvanik/xenia/commit/70bd283e80125932339f23a54e5efcb3c74f0456)

# Issues:
- there is audio garbage
- the menu's fonts are corrupted
- the content saving is not possible without it can't continue game testing. (always put back to saving contents on menu)

# Log:
[GoldenAxeBR_70bd283e_161210.zip](https://github.com/xenia-project/game-compatibility/files/644067/GoldenAxeBR_70bd283e_161210.zip)

# Labels:
- state-menu
- gpu-corrupt-drawing
- gpu-missing-drawing

**Comment by Sleepy93** [comment]
# Update
update the logs, upload a screen.
problem with saving game.

# Screen
![golden](https://cloud.githubusercontent.com/assets/20132031/21956927/65a5df62-da8b-11e6-8f36-78a92cb3fdfa.png)

# Tested 
On (https://github.com/benvanik/xenia/commit/39cb957f1b827bc7f991dc19ee5f7a6f25aa50fb)

# Log file

[GABR_39cb957f_menus_gpu_audio_save.txt](https://github.com/xenia-project/game-compatibility/files/706116/GABR_39cb957f_menus_gpu_audio_save.txt)

# Labels
- state-menus
- audio-garbage
- gpu-missing-drawing
- gpu-corrupt-drawing

**Comment by Sleepy93** [comment]
# Update
> w> 00000004 GPU: Write to unknown register (1973 = 00010000)
GPU: Write to unknown register a lot of times..

# Tested
On (https://github.com/benvanik/xenia/commit/761ae4eeeecf9a993ca6e0075b6d601b63d7deb8)

# Log file
[xenialog.txt](https://github.com/xenia-project/game-compatibility/files/926223/xenialog.txt)

# Labels
(still)
- state-menus
- gpu-missing-drawing
- gpu-corrupt-drawing

**Comment by Sleepy93** [comment]
# Update
w> 00000004 GPU: Write to unknown register (0081 = 80010000)
w> 00000004 GPU: Write to unknown register (0082 = 00000000)
w> 00000004 GPU: Write to unknown register (1973 = 00010000)
w> 00000004 GPU: Write to unknown register (1964 = 00000001)

# Tested 
On(https://github.com/benvanik/xenia/commit/e8f55d8ff276cce22cf7c44fa95c2fac974ddbef)

# Log file
[GoldenAxe_log.txt](https://github.com/xenia-project/game-compatibility/files/1036577/GoldenAxe_log.txt)

# Labels
- gpu-corrupt-drawing
- gpu-missing-drawing
- kernel-save-file-errors
- state-menus

**Comment by Parovozik** [comment]
Tested on (https://github.com/benvanik/xenia/commit/2a52156edf4ecc02d9083c31d495e0a927d26438)
# Log:
[Golden Axe.log](https://github.com/xenia-project/game-compatibility/files/1772963/Golden.Axe.log)
# Issues:
Boot - Ok, but trouble on main screen with save state. Status - main screen. Broken fonts.
# Screenshots:
![bandicam 2018-03-02 00-30-02-658](https://user-images.githubusercontent.com/19775644/36872489-8321e6ca-1db6-11e8-94e8-541405126dfd.jpg)
![bandicam 2018-03-02 00-30-03-625](https://user-images.githubusercontent.com/19775644/36872493-862798d8-1db6-11e8-8d44-39122c0bdbb3.jpg)

**Comment by Parovozik** [comment]
Tested on (https://github.com/xenia-project/xenia/commit/2314f25fb7252bb9054fd7c863e1a57bee90282d)
Still error "storage device not found", but graphics is fixed
![bandicam 2018-06-07 09-51-10-739](https://user-images.githubusercontent.com/19775644/41116286-8972bfe4-6a92-11e8-82d1-8025b678bb35.jpg)
![bandicam 2018-06-07 09-51-56-827](https://user-images.githubusercontent.com/19775644/41116292-8bcaf752-6a92-11e8-9845-4bbe7a334f23.jpg)
# Labels
kernel-save-file-errors
state-menus

**Comment by itornukem64** [comment]
Tested on (xenia-project/xenia@e6068e0d)
Reaches in game however xenia crashes constanstly (after the cinematic where the dragon is summoned)  I was able to reach the firts auto save place so the save feature is now working correctly and there's also some corrupt drawing related to FOV or DOF I don't know only the ground aound the character is drawing correctly
![Xenia_master280320 Screenshot 2020 03 28 - 18 00 43  #93](https://user-images.githubusercontent.com/12883686/79771092-8942c000-82eb-11ea-8a83-97bb7434ddbb.png)

**Comment by Margen67** [comment]
@itornukem64 Please provide a log.

**Comment by itornukem64** [comment]
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/4506223/xenia.zip)


> @itornukem64 Please provide a log.

Here is it

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/74cb0117cf616c8f8a4d56d1ca6fdc59a42d1813

Completed first challenge. 120+FPS with vsync off and shaders cached. Saves work.

![gaxx](https://user-images.githubusercontent.com/31154606/103261664-e48f7c80-4967-11eb-9d8c-32d58007ebe9.png)

<details>
  <summary>More Screenshots</summary>

![gax (1)](https://user-images.githubusercontent.com/31154606/103261490-6501ad80-4967-11eb-9c0a-f268749d4680.png)
![gax (2)](https://user-images.githubusercontent.com/31154606/103261491-659a4400-4967-11eb-9131-43176fc6e6b0.png)
![gax (3)](https://user-images.githubusercontent.com/31154606/103261493-6632da80-4967-11eb-9d51-350a5b1f169c.png)
![gax (4)](https://user-images.githubusercontent.com/31154606/103261494-6632da80-4967-11eb-97e2-d3fe96d45f85.png)
![gax (5)](https://user-images.githubusercontent.com/31154606/103261480-603cf980-4967-11eb-8e91-4a612563e2a6.png)
![gax (6)](https://user-images.githubusercontent.com/31154606/103261495-66cb7100-4967-11eb-8336-c89fe13ff7b6.png)

</details>

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5749711/xenia.zip)

state-gameplay | tech-middleware-havok

**Comment by itornukem64** [comment]
> Tested on [xenia-project/xenia@74cb011](https://github.com/xenia-project/xenia/commit/74cb0117cf616c8f8a4d56d1ca6fdc59a42d1813)
> 
> Completed first challenge. 120+FPS with vsync off and shaders cached. Saves work.
> 
> ![gaxx](https://user-images.githubusercontent.com/31154606/103261664-e48f7c80-4967-11eb-9d8c-32d58007ebe9.png)
> 
> More Screenshots
> [xenia.zip](https://github.com/xenia-project/game-compatibility/files/5749711/xenia.zip)
> 
> state-gameplay | tech-middleware-havok

Yeah beeen playing the game for a while now but I'm to lazy to report, also please add gpu-missing-drawing because the ground that is "not-so close" to the character does not render properly
